tests/file/test_handle.py

```diff
@@ -0,0 +1,153 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import unittest
+
+from azure.storage.file import (
+    FileService,
+)
+from tests.testcase import (
+    StorageTestCase,
+    record,
+    TestMode,
+)
+
+# ------------------------------------------------------------------------------
+TEST_SHARE_NAME = 'test'
+
+
+# ------------------------------------------------------------------------------
+
+class StorageHandleTest(StorageTestCase):
+    def setUp(self):
+        super(StorageHandleTest, self).setUp()
+        self.fs = self._create_storage_service(FileService, self.settings)
+
+    def tearDown(self):
+        return super(StorageHandleTest, self).tearDown()
+
+    def _validate_handles(self, handles):
+        # Assert
+        self.assertIsNotNone(handles)
+        self.assertGreaterEqual(len(handles), 1)
+        self.assertIsNotNone(handles[0])
+
+        # verify basic fields
+        # path may or may not be present
+        # last_connect_time_string has been missing in the test
```

Comment:
We should still test that we can parse these out if they are present

Reply:
The condition to return last_connect_time_string is too hard to simulate. I verified that path is getting deserialized correctly.